__tests__/app.test.js

```diff
@@ -187,3 +187,55 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("returns 200 status code", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("returns an array of comments for the article id with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        const comments = body.comments;
+        comments.forEach((comment) => {
+          expect(typeof comment).toBe("object");
```

Comment:
It would be beneficial to include the article id here and specify which id you expect, to ensure its the right comments being returned